src/util.ts

```diff
@@ -126,7 +126,11 @@ let telemtryReporter: TelemetryReporter;
 let toolsGopath: string;
 
 export function getGoConfig(): vscode.WorkspaceConfiguration {
-	return vscode.workspace.getConfiguration('go', vscode.window.activeTextEditor ? vscode.window.activeTextEditor.document.uri : null);
+	return vscode.window.activeTextEditor ? getGoConfigForUri(vscode.window.activeTextEditor.document.uri) : null;
+}
+
+export function getGoConfigForUri(uri: vscode.Uri): vscode.WorkspaceConfiguration {
```

Comment:
Instead of a separate function that takes uri, you can tweak the original utility to take an optional param

```javascript
export function getGoConfig(uri?: vscode.Uri): vscode.WorkspaceConfiguration {
     if (vscode.window.activeTextEditor) {
        uri = vscode.window.activeTextEditor.document.uri;
     }
    return vscode.workspace.getConfiguration('go', uri);
}
```